src/Summary.jsx

```diff
@@ -132,6 +134,11 @@ class Summary extends React.Component {
               </Link>
               <h1>Sobre el área</h1>
               <div className="line" />
+              <h5 className="hectareas vulne">
+                Vulnerabilidad:
+                {' '}
+                <b>0,33</b>
```

Comment:
This value should be get it from the API response, and in case it's  null or undefined show a message

Reply:
Created "activeArea.vulnerability" as a parameter to add in the endpoint "/anh_areas/${name}"